tests/factory/conftest.py

```diff
@@ -0,0 +1,8 @@
+import pytest
+from brownie import ModuleFactory
+
+
+@pytest.fixture()
+def module_factory(accounts):
+    factory = ModuleFactory.deploy({"from": accounts[8]})
```

Comment:
why `[8]`?

Reply:
no specific, random number picked, since those no matter who deploys it